test/scenarios/bookshop/update.test.js

```diff
@@ -24,6 +24,24 @@ describe('Bookshop - Update', () => {
     expect(res.data.author_ID).to.be.eq(201)
     expect(res.data.descr).to.be.eq('UPDATED')
   })
+  test('Update array of', async () => {
+    // create book
+    const insert = INSERT.into('sap.capire.bookshop.Books')
+      .columns(['ID', 'footers'])
```

Comment:
Should `footers` be `footnotes`?

Reply:
thanks for catching that. It seems like the inital creation of the `footnotes` was not even necessary to resproduce the issue.